src/ast/types.rs

```diff
@@ -61,6 +61,7 @@ pub enum BindingKind {
 pub struct Binding {
     pub kind: BindingKind,
     pub location: Location,
+    pub end_location: Location,
```

Comment:
very exciting!

Reply:
I don't know if the spans will always be "correct" (that is: I can't guarantee they'll match the CPython semantics), so we may need to iterate on them a bit.

Reply:
(These fields are now available on `Message` and `Check`.)